README.md

```diff
@@ -1,13 +1,11 @@
 # Survey form with React
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+A form taking in input displayed to the user when clicking submit.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+First time using state in React! Started with creating a form and style it, thereafter approached the part of connecting the state values to the user summary.
 
```

Comment:
i had the same approach! What do you think, was it the best way to start of the project?